tests/adapters/shell/terminal_client_test.py

```diff
@@ -198,6 +226,32 @@ class TelnetClientTest(TerminalClientTest):
     client = TelnetClient
     port = 10011
 
+    @patch('netman.adapters.shell.telnet._connect')
+    @patch('netman.adapters.shell.telnet.TelnetClient._login', Mock())
+    def test_changing_default_connect_timeout(self, connect_method_mock):
+        TelnetClient(**self._get_some_credentials())
+        self.assertEqual(60, connect_method_mock.call_args[0][2])
+
+        TelnetClient.set_default_connect_timeout(120)
+
+        TelnetClient(**self._get_some_credentials())
+        self.assertEqual(120, connect_method_mock.call_args[0][2])
+
+        TelnetClient.set_default_connect_timeout(60)
+
+    @patch('netman.adapters.shell.telnet._connect', Mock())
+    @patch('netman.adapters.shell.telnet.TelnetClient._login', Mock())
+    def test_changing_default_command_timeout(self):
+        telnet = TelnetClient(**self._get_some_credentials())
+        self.assertEqual(300, telnet.command_timeout)
+
+        TelnetClient.set_default_command_timeout(600)
+
+        telnet2 = TelnetClient(**self._get_some_credentials())
+        self.assertEqual(600, telnet2.command_timeout)
+
+        TelnetClient.set_default_command_timeout(300)
```

Comment:
I have to change back the timeout to a default after the test ... I don`t like this.